submissions/krylenger/html-css-popup/style.css

```diff
@@ -0,0 +1,245 @@
+}
+
+.navigation-container__body {
+    width: 320px;
+    height: 100%;
+}
+
+.pop-up-container {
+    display: none;
+    text-align: center;
+    height: 420px;
+    width: 270px;
+    border: solid 1px lightgrey;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px 0px rgba(0,0,0,1);
+}
+
+.pop-up-icons-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: center;
+    height: 94%;
+}
+
+ul {
+    display: flex;
+    justify-content: space-around;
+    margin: 0;
+    padding-inline-start: 0px;
+}
+
+ul li {
+    list-style-type: none;
+    text-align: left;
+}
+
+.navigation-container__list-of-items > li {
+    width: 20%; 
+    display: flex;
+    justify-content: center;
+}
+
+.toggle-pop-up:checked ~ .toggle-pop-up-label::before {
+    position: absolute;
+    content: "";
+    right: 153px;
+    top: 53px;
+    width: 10px;
+    height: 10px;
+    background-color: white;
+    transform: rotate(45deg);
+    border-top: 2px solid grey;
+    border-left: 2px solid grey;
+    z-index: 1;
+}
+
+.navigation-container__list-of-items a , .toggle-pop-up-label {
+    color: black;
+    line-height: 50px;
+}
+
+.toggle-additional-icons {
+    position: absolute;
+    bottom: 0px;
+    left: 0px;
+    opacity: 0;
+}
+
+.toggle-additional-icons:focus + label, .link-to-kottans:focus {
+    background-color: rgba(0, 197, 0, 0.5);
+}
+
+
+.toggle-additional-icons-label {
+    width: 100%;
+    height: 6%;
+    text-align: center;
+    font-size: 20px;
+    font-weight: 100;
+    background-color: rgba(197, 197, 197, 0.4);
+}
+
+
+.toggle-pop-up:checked ~ .pop-up-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: flex-start;
+    position: absolute;
+    right: 20px;
+    top: 60px;
+    overflow: auto;
+}
+
+.item {
+    width: 80px;
+    height: 80px;
+    margin: 2px 2px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    z-index: 2;
+}
+
+.item:hover {
+    background-color: rgba(0, 197, 0, 0.5);
+    border-radius: 5px;
+}
+
+.item:focus {
+    outline: 2px solid lightgreen;
+    box-shadow: 0 0 3pt 2pt green;
+}
+
+.pop-up-icons-container-additional {
+    display: none;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-top: 40px;
+}
+
+.toggle-additional-icons:checked ~ .pop-up-icons-container-additional {
+    display: flex;
+    flex-wrap: wrap;
+    height: 40px;
+}
+
+.toggle-additional-icons:checked ~ .toggle-additional-icons-label {
+    display: none;
+}
+
+.toggle-additional-icons:checked ~ .pop-up-icons-container-additional:before {
+    content: '';
+    width: 100%;
+    position: absolute;
+    top: 400px;
+    left: 0;
+    border-bottom: 1px solid black;
+}
+
+.link-to-kottans {
+    margin-top: 20px ;
+    width: 100%;
+    background-color: rgba(197, 197, 197, 0.4);
+}
+
+.link-to-kottans a {
+    line-height: 30px;
+}
+
+.link-to-kottans a {
```

Comment:
You can combine these styles with the previous selector

Reply:
✅ done